### PR TITLE
German language and spelling corrections

### PR DESCRIPTION
**Pull request by Racooder**
Language and spelling corrections for the German translation of the menu and manual.
Open for edits.

### CODE REVIEW

File: Assets/Data/Localization/German/Manual.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-Oben auf deinem Bildschirm werden drei Standorte angezeigt, die jeweils ein Heißluftballon haben. Fliege dein Flugzeug in eins der Heißluftballons, um ein Paket abzuholen.
+Oben auf deinem Bildschirm werden drei Standorte angezeigt, an denen jeweils ein Heißluftballon zu finden ist. Fliege dein Flugzeug in einen der Heißluftballons, um ein Paket abzuholen.
 
-Nachdem ein paket abgeholt wurde, ändert sich der Standort oben auf deinem Bildschirm zu einem Lieferort. Fliege über die Stadt (oder deine beste Vermutung wo sie ist) und lasse das Paket fallen. Du wirst Boost Punkte bekommen, je nachdem wie weit das Paket von dem beabsichtigten Ziel weg ist. Boost Punkte ermöglichen dir, für eine begrenzte Zeit schneller zu fliegen.
+Nachdem ein Paket abgeholt wurde, ändert sich der Standort oben auf deinem Bildschirm zu einem Lieferort. Fliege über die Stadt (oder deine beste Vermutung wo sie ist) und lasse das Paket fallen. Du wirst, je nachdem wie weit das Paket von dem vorgesehenen Ziel entfernt ist, Boost Punkte erhalten. Boost Punkte ermöglichen dir, für eine begrenzte Zeit schneller zu fliegen.
```
Comment (jancborchardt) [inline Assets/Data/Localization/German/Manual.txt:5]:
```suggestion
Nachdem ein Paket abgeholt wurde, ändert sich der Standort oben auf deinem Bildschirm zu einem Lieferort. Fliege über die Stadt (oder deine beste Vermutung wo sie ist) und lasse das Paket fallen. Du wirst, je nachdem wie weit das Paket von dem vorgesehenen Ziel entfernt ist, Boost-Punkte erhalten. Boost-Punkte ermöglichen dir, für eine begrenzte Zeit schneller zu fliegen.
```

File: Assets/Data/Localization/German/MenuLocalization.json [hunk 2]
```diff
@@ -126,19 +126,19 @@
...
         },
         {
             "id": "settingsMenu.controls.extraControls",
-            "text": "Extra Steuerungen"
+            "text": "Weitere Steuerungen"
         },
         {
             "id": "settingsMenu.controls.makeDayTime",
-            "text": "Setze zu Tag"
+            "text": "Mach es Tag"
```
Comment (jancborchardt) [inline Assets/Data/Localization/German/MenuLocalization.json:137]:
```suggestion
            "text": "Tageszeit"
```
Sounds a bit nicer

File: Assets/Data/Localization/German/MenuLocalization.json [hunk 3]
```diff
@@ -126,19 +126,19 @@
...
         },
         {
             "id": "settingsMenu.controls.makeNightTime",
-            "text": "Setze zu Nacht"
+            "text": "Mach es Nacht"
```
Comment (jancborchardt) [inline Assets/Data/Localization/German/MenuLocalization.json:141]:
```suggestion
            "text": "Nachtzeit"
```
Sounds a bit nicer too

File: Assets/Data/Localization/German/MenuLocalization.json (hunk 2 above)
Comment (Racooder) [reply Assets/Data/Localization/German/MenuLocalization.json:137]: Maybe that's not clear enough. I thought about that too.

File: Assets/Data/Localization/German/MenuLocalization.json (hunk 3 above)
Comment (Racooder) [reply Assets/Data/Localization/German/MenuLocalization.json:141]: Same here, maybe not clear enough

Comment (MillersMan) [reply Assets/Data/Localization/German/MenuLocalization.json:141]: In my first translation try I had
```suggestion
            "text": "Zu Nacht wechseln"
```
In my opinion it sounds bettern tahn "Mach es Nacht" but its still not perfect

File: Assets/Data/Localization/German/MenuLocalization.json (hunk 2 above)
Comment (MillersMan) [reply Assets/Data/Localization/German/MenuLocalization.json:137]: In my first translation try I had
```suggestion
            "text": "Zu Tag wechseln"
```

In my opinion it sounds better than "Mach es Tag" but its still not perfect

Comment (Racooder) [reply Assets/Data/Localization/German/MenuLocalization.json:137]: I don't really know which one sounds better.

File: Assets/Data/Localization/German/MenuLocalization.json (hunk 3 above)
Comment (Racooder) [reply Assets/Data/Localization/German/MenuLocalization.json:141]: Same her I don't really know which one sounds better.

File: Assets/Data/Localization/German/MenuLocalization.json (hunk 2 above)
Comment (jancborchardt) [reply Assets/Data/Localization/German/MenuLocalization.json:137]: "Zu Tageszeit wechseln" / "Zu Nachtzeit wechseln"?

Or " zu" → "auf" possibly. But there's usually a better word than "machen". :)

Comment (Racooder) [reply Assets/Data/Localization/German/MenuLocalization.json]: > "Zu Tageszeit wechseln" / "Zu Nachtzeit wechseln"?
> 
> Or " zu" → "auf" possibly. But there's usually a better word than "machen". :)

In my opinion the best solution. I changed it.

**Comment by SebLague** [comment]
Thanks for the improvements!

**Pull request merged**